lib/seeing_is_believing/evaluate_by_moving_files.rb

```diff
@@ -83,68 +87,72 @@ def set_back_to_initial_conditions
     end
 
     def write_program_to_file
-      File.open(filename, 'w') { |f| f.write program.to_s }
+      File.open(filename, 'w', external_encoding: "utf-8") { |f| f.write program.to_s }
     end
 
     # have to basically copy a bunch of Open3 code into here b/c keywords don't work right when the keys are not symbols
     # https://github.com/ruby/ruby/pull/808    my PR
     # https://bugs.ruby-lang.org/issues/10699  they opened an issue
     # https://bugs.ruby-lang.org/issues/10118  weird feature vs bug conversation
     def evaluate_file
+      event_server = TCPServer.new(port)
```

Comment:
Made it a low-importance issue on https://github.com/JoshCheek/seeing_is_believing/issues/94